### PR TITLE
69 Update data with `adae` changes

### LINKED ISSUE #69: Add variables AESCONG, AESDISAB, AESHOSP, AESLIFE, and AESMIE to adae
### Feature description

Updated ADAE in random.cdisc.data: https://github.com/insightsengineering/random.cdisc.data/pull/193

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct.

### Contribution Guidelines

- [X] I agree to follow this project's Contribution Guidelines.

### Security Policy

- [X] I agree to follow this project's Security Policy.

### PR DESCRIPTION
**Pull request by edelarua**
Closes #69

### CODE REVIEW

File: data_raw/rcd.R [hunk 1]
```diff
@@ -5,7 +5,7 @@ releases <- c(
   # "2022_01_28" = "7f2200332aab693d2ef4149a0f67b7708a2db516",
   # "2022_02_28" = "fc3620365f38c36e45880cbdebfe16508d902fa6",
   # nolint end
-  "2022_06_27" = "c1a972e7a177eb4556aa01e6a54d27d6b4d32782"
+  "2022_06_27" = "33ab12d00478c72b74d82150dbbd748af196dccf"
```
Comment (codesformochi) [inline data_raw/rcd.R:8]: is this the right commit? i checked the logs of the RCD/main for the issue and it has a different commit in the log (please let me know if i'm looking at the log wrong).
Screen Shot 2022-09-08 at 5 19 24 PM

Comment (codesformochi) [reply data_raw/rcd.R:8]: this is my only comment! the rest looks good!

**Review by codesformochi — CHANGES_REQUESTED** [review]

**Comment by shajoezhu** [comment]
Hi  @edelarua @codesformochi , is this good to go? :)

**Comment by edelarua** [comment]
> Hi @edelarua @codesformochi , is this good to go? :)

@shajoezhu this should be good to go, but we could add in the updates to adsl as well if we wait and merge https://github.com/insightsengineering/random.cdisc.data/pull/200 first.

**Comment by shajoezhu** [comment]
Brilliant! Thanks @edelarua ! ok, let's wait for https://github.com/insightsengineering/random.cdisc.data/pull/194 as well then. we can check in on this again tonight

**Comment by shajoezhu** [comment]
hi @edelarua , would you mind do another update on this please

image

https://github.com/insightsengineering/random.cdisc.data/commits/main

to e96be720f5828bbe869325e9fc15e227f998f3dd

**Review by shajoezhu — APPROVED** [review]
lgtm! Thanks @edelarua

**Pull request merged**